Standards/scs-0214-v1-k8s-node-distribution.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Kubernetes Node Distribution and Availability
+type: Standard
+status: Draft
+track: KaaS
+---
+
+## Introduction
+
+A Kubernetes instance is provided as a cluster, which consists of a set of machines,
+so called nodes. A cluster is composed of a control plane and at least one worker node.
+The control plane manages the worker nodes and therefore the pods in the cluster by making
+decisions about scheduling, event detection and rights management. Inside the control plane,
+multiple components exist, which can be duplicated and distributed over multiple machines
```

Comment:
IMHO we should speak of nodes most of the times until the reader reaches glossary when describing components of an Kubernetes cluster. It may be provided by physical machines or machine flavors on hyperscalers but may also be VM instances also named "instance" instead of a "machine" provided by a classical IaaS. The glossary follows below, so at this time "machine" is fine but still I personally like the k8s name "node" more.

Reply:
I will write `node` in most of this section now, but just to write why I choose `machine`. This was based on the Kubernetes documentation [here](https://kubernetes.io/docs/concepts/architecture/nodes/), which states physical or virtual machines as the base of a node.